src/components/story-circle/index.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import * as s from './styles';
+import { WaveclassTheme } from '../../layout';
+
+import { StoryCircleProps } from './types';
+import { Avatar } from '../avatar';
+
+export const StoryCircle = ({
+	profilePictureUrl,
+	username,
+	highlight,
+}: StoryCircleProps): JSX.Element => {
```

Comment:
Precisaremos de mais uma propriedade aqui: onClick => void que deve ser chamada quando o StoryCircle for clicado